src/patternfly/components/Radio/radio.scss

```diff
@@ -6,26 +6,24 @@
   --#{$radio}__label--Color: var(--#{$pf-global}--Color--100);
   --#{$radio}__label--FontWeight: var(--#{$pf-global}--FontWeight--normal);
   --#{$radio}__label--FontSize: var(--#{$pf-global}--FontSize--md);
-  --#{$radio}__label--LineHeight: var(--#{$pf-global}--LineHeight--sm);
-  --#{$radio}__input--Height: var(--#{$radio}__label--FontSize);
-  --#{$radio}__input--MarginTop: calc(((var(--#{$radio}__label--FontSize) * var(--#{$radio}__label--LineHeight)) - var(--#{$radio}__input--Height)) / 2);
+  --#{$radio}__label--LineHeight: var(--#{$pf-global}--LineHeight--md);
+  --#{$radio}__input--TranslateY: #{pf-size-prem(5px)}; // FF needs an offset
   --#{$radio}__input--first-child--MarginLeft: #{pf-size-prem(1px)};
   --#{$radio}__input--last-child--MarginRight: #{pf-size-prem(1px)};
   --#{$radio}__description--FontSize: var(--#{$pf-global}--FontSize--sm);
   --#{$radio}__description--Color: var(--#{$pf-global}--Color--200);
   --#{$radio}__body--MarginTop: var(--#{$pf-global}--spacer--sm);
 
-  display: grid;
+  display: inline-grid;
```

Comment:
Same comment here as the checkbox - it's expected this component will have normal block formatting.